src/styles/components/_create-page__form.css

```diff
@@ -0,0 +1,46 @@
+.create-page__form {
+  max-width: 360px;
+  padding: 18px;
+  margin: 0 auto;
+  display: relative;
+}
+::placeholder {
+  /* Chrome, Firefox, Opera, Safari 10.1+ */
+  color: var(--color--grey);
+  opacity: 1; /* Firefox */
+}
+
+.create-page__form__heading {
+  color: whitesmoke;
+  font-size: 1em;
+  font-weight: bold;
+  margin-block-end: 0.1em;
+  text-shadow: 2px 1px 3px black;
+}
+
+.create-page__form__label {
+  display: block;
+}
+
+.create-page__form__add-answer__counter,
+.create-page__form__add-question__counter,
+.create-page__form__add-tags__counter {
+  position: relative;
+  font-size: 0.7em;
+  color: var(--color--grey);
+  top: -28px;
+  left: 297px;
+  width: 19px;
+  height: 0;
```

Comment:
Why is that needed?

Reply:
I set the height because the div shouldn't use up space in its place of origin when I moved it into the textarea and the width is set to 19px so I can properly align the counters to be even, whether 2 or 3 digits are on display.

Reply:
You should use `position: absolute` for the counters so they don't use up space. The width of `19px` also sounds a bit _hacky_. Isn't aligning them (maybe to the right) enough?